src/main/docker/root/docker-entrypoint.sh

```diff
@@ -18,4 +18,13 @@ fi
 mkdir --parents /root/.config/patroni
 ln -s /config/patronictl.yaml /root/.config/patroni/patronictl.yaml
 
-exec gosu postgres /patroni.py /config/patroni.yml
+
+if [[ -d "/config/patroni" ]]; then
```

Comment:
maybe is better to switch the 2 conditions? 

first check if patroni.yaml file exist and if not switch to config directory?

because in this case, if one would use a patroni.yml and mount it in /config/patroni.yml it would not be used since we by default create the /config/patroni directory in the image ...

or document that any patroni config files/directory on this image should be mounted on /config instead